build_tools/benchmarks/mmperf/run_mmperf.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Runs SHARK tank using both SHARK-Runtime and IREE-Runtime, producing benchmark
+# numbers.
+#
+# Usage:
+#    ./run_shark.sh \
+#        <SHA of https://github.com/nod-ai/SHARK.git to pin to> \
+#        <pytest regex> e.g. "cpu", "cuda", "cuda and torch".
+#        <driver> e.g. "cpu", "cuda", "vulkan"
+#        <output directory>
+
+set -xeuo pipefail
+
+export REPORT_DIR=$1
+# Either `cpu` or `cuda`.
+export BACKEND=$2
+
+git clone https://github.com/mmperf/mmperf.git
+cd mmperf
+
+# Update IREE.
+cd external/iree
+git fetch --all
+git checkout origin/main
+git submodule update --init --recursive
+cd -
+
+# Create virtual environment.
+python3 -m venv mmperf.venv
+source mmperf.venv/bin/activate
+pip install -r requirements.txt
+pip install -r ./external/llvm-project/mlir/python/requirements.txt
+pip install requests
```

Comment:
Here's `requests` again ([discussion on Discord yesterday](https://discord.com/channels/689900678990135345/689957613152239638/1036775952832807032))... I saw this got added to a few requirements files, so you might not need to explicitly install it here.

Reply:
Removed